MojioSDK/Models/Accounts/User.swift

```diff
@@ -48,8 +48,8 @@ public struct User: UserModel {
     public let phoneNumbers: [P]
     public let tags: [String]
     public let defaultLanguage: String?
-    public let createdOn: Date?
-    public let lastModified: Date?
+    public let createdOn: String?
+    public let lastModified: String?
```

Comment:
Why type was changed to `String` ? `User` is one of base structures so this change will impact a lot of app code.

Reply:
@alex-kepar Thanks for the highlight. We need to implement a few Date operation in our application and due to that, I am getting an error while parsing the JSON into Model that's the reason I change the Date type into String.  I requested to the @davidjed  to create a new branch for all changes because I have added list of features in this release.

Reply:
Ok. Thanks @shahbuddin-pb for clarification.
I just see that this PR is made for "phoenix" branch. So I would like highlight that these changes will broke "Phoenix" and "Force" applications.

Reply:
Hmm yeah I definitely don't want to break existing apps. Need to find a way to make these compatible with both.